consolidate redundant office hours documentation (#2923)
astronomer/docs: astro/office-hours.md

> Before you book an office hour appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that answer your question. 

Reviewer: ```suggestion
Before you book an office hours appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that answer your question. 
```

> To book office hours, click **Book office hours** in the left sidebar of the Astronomer documentation site. This will take you to a Calendly form to schedule a 30-minute virtual meeting. In the form, provide details about any issues and questions that you want to discuss during the meeting.

Reviewer: ```suggestion
To book office hours, click **Book office hours** in the documentation sidebar navigation. Then, schedule a 30-minute virtual meeting on the Calendly form. In the form, provide details about any issues and questions that you want to discuss during the meeting.
```
- This page is in the documentation site, so we can assume users don't require that detail. 
- Spatial organization (left right top bottom above below) isn't a helpful cue for screen reader users, so we need a different term.
